src/main/kotlin/com/zepben/evolve/cim/iec61970/base/core/Terminal.kt

```diff
@@ -42,12 +39,6 @@ class Terminal @JvmOverloads constructor(mRID: String = "") : AcDcTerminal(mRID)
     var normalFeederDirection: FeederDirection = FeederDirection.NONE
     var currentFeederDirection: FeederDirection = FeederDirection.NONE
 
-    @Deprecated(
-        message = "Replaced by direct use of `normalPhases` and `currentPhases`.",
-        replaceWith = ReplaceWith("/* See [TracedPhases.normal] and [TracedPhases.current] for direct replacement suggestions */"),
-    )
-    val tracedPhases: TracedPhases get() = TracedPhases(terminal = this)
```

Comment:
Deleted this as the branch has so many breaking changes removing deprecated stuff just seemed sensible.

Reply:
Yep good call. I have added it.